run_desktop_tests.sh

```diff
@@ -54,7 +54,7 @@ echo "Running Clippy lints..."
 cargo clippy --lib --tests --bins --benches --features std -- -D warnings
 cargo clippy --lib --tests --bins --benches --features std,"$MOST_FEATURES" -- -D warnings
 (cd libraries/opensk && cargo clippy --features std -- -D warnings)
-(cd libraries/opensk && cargo clippy --features std,config_command,debug_ctap,with_ctap1,vendor_hid,ed25519 -- -D warnings)
+(cd libraries/opensk && cargo clippy --features std,config_command,debug_ctap,with_ctap1,vendor_hid,ed25519,fingerprint -- -D warnings)
```

Comment:
What about `std,"$MOST_FEATURES"` like 2 lines above?

Reply:
That doesn't work since these are features from the base Cargo.toml, and here, clippy is running inside `libraries/opensk`. But I could use `--all-features`, and the only difference would be `fuzz`. I don't remember why I explicitly omitted `fuzz` when I wrote this. But it works either way.